src/pages/[...params].js

```diff
@@ -14,10 +14,11 @@ const Fallback = () => {
       router.push('/login/');
     }
 
-    delete query.params;
+    const queryParameters = { ...query };
+    delete queryParameters.params;
```

Comment:
This change (also) belongs in #123 . Maybe you can cherry-pick the commit with this change to that PR as well?

Reply:
Note also my comment on #123 to add a comment here to explain why we remove the `.params`